website/docs/components/button/button-a11y.md

```diff
@@ -22,35 +22,24 @@ The list below describes roles and attributes that component already has.
 
 Table: Roles and attributes
 
-| Role     | Attribute              | Element    | Usage                                                                                                                                       |
-| -------- | ---------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
-| `button` |                        | `div`, `a` | Identifies the element as a button. Accessible name for the button is defined by the text content of the element or by adding `aria-label`. |
-|          | `tabIndex="0"`         | `div`, `a` | Includes the element in the `Tab` sequence. Needed on the `a` element because it doesn't have a `href` attribute.                          |
-|          | `aria-pressed="false"` | `a`        | Identifies the button as a toggle button. Indicates the toggle button isn’t pressed.                                                       |
-|          | `aria-pressed="true"`  | `a`        | Identifies the button as a toggle button. Indicates the toggle button is pressed.                                                           |
+| Component / element | Attribute                                               | Usage                                                                                                                                                                   |
```

Comment:
I think we can remove the Component column on this page, the component is simple enough